src/public/interfaces.ts

```diff
@@ -364,6 +364,11 @@ export interface StartConversationRequest {
   */
   title: string;
 
+  /**
+  * The Id of the conversation
```

Comment:
NIT: Add a comment indicating that this is optional and should be specified whenever a previous conversation about a specific sub-entity has already been started before